css/addOrEditTask.css

```diff
@@ -0,0 +1,250 @@
+@import "./normalize.css";
+@import "./variables.css";
+
+
+  .new-task-box {
+    height: 100%; 
+    background-color: var(--white);
+    border-radius: var(--small);
+    box-shadow: var(--primaryShadowColor);
+    overflow-y: scroll;
+    overflow-x: hidden;
+  }
+
+  .new-task-box::-webkit-scrollbar-track {
+	  border-radius: var(--small);
+  }
+
+  .new-task-box::-webkit-scrollbar {
+	  width: var(--small);
+  }
+
+  .new-task-box::-webkit-scrollbar-thumb{ 
+	  border-radius: var(--small);
+	  background-color: var(--extraColor);
+  }
+  
+  form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: var(--medium);
+  }
+  
+  label {
+    font-size: larger;
+    margin: var(--large) 0 var(--medium) 0;
+  }
+  
+  .input {
+    font-size: medium;
+    padding: var(--small);
+    border: none;
+    outline: none;
```

Comment:
I personally like `reset.css`. The reason is that you can avoid doing such things.